data/sql/updates/pending_db_world/rev_1696800156876281900.sql

```diff
@@ -0,0 +1,568 @@
+--
+-- Remove existing Durotar Herbalism Nodes:
+DELETE FROM pool_gameobject WHERE pool_entry BETWEEN 498 AND 501;
```

Comment:
```suggestion
DELETE FROM `pool_gameobject` WHERE `pool_entry` BETWEEN 498 AND 501;
```
Oof, they're gone again.